scripts/check_tx.py

```
@@ -0,0 +1,177 @@
+            self.driver.get(url)
+            time.sleep(5)
+            
+            all_elements = self.driver.find_elements(By.XPATH, "//*[text()]")
+            all_text = [elem.text for elem in all_elements]
+            
+            # Extract execution resources
+            resources = {}
+            resources_started = False
+            current_number = None
+            
+            for i, text in enumerate(all_text):
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

(B007)